test/e2e/framework.go

```diff
@@ -1190,6 +1192,31 @@ func (b *PodBuilder) WithMutateFunc(f func(*corev1.Pod)) *PodBuilder {
 	return b
 }
 
+func (b *PodBuilder) AddVolume(volume []corev1.Volume) *PodBuilder {
+	b.Volumes = volume
+	return b
+}
+
+func (b *PodBuilder) AddVolumeMount(volumeMount []corev1.VolumeMount) *PodBuilder {
+	b.VolumeMounts = volumeMount
+	return b
+}
+
+func (b *PodBuilder) MountConfigMap(configMapName string, mountPath string) *PodBuilder {
+	volumeMount := corev1.VolumeMount{Name: "config-volume", MountPath: mountPath}
+	volume := corev1.Volume{
+		Name: "config-volume",
```

Comment:
I'd suggest making the volume name configurable (with a function parameter) rather than hardcoding it to `"config-volume"`. I don't expect `MountConfigMap` to be called multiple times for a given Pod, but still.